src/commands/txCommands/util/index.js

```diff
@@ -63,7 +63,9 @@ const reduceUniques = localTrns => {
 const extractTrnSource = () =>
 	glob
 		.sync(
-			`${paths.repoRoot}/src/+(components|app)/**/!(*.test|*.story).jsx`
+			`${
+				paths.repoRoot
+			}/src/+(components|app|route-*)/**/!(*.test|*.story).jsx`
```

Comment:
wonder if we should just change this to `/src/**/!(*.test|*.story).jsx` so that we don't have to keep updating this as we experiment with different naming conventions. Or, we can start aiming toward Lerna now and make it

```
/(src|packages)/**/!(*.test|*.story).jsx
```